packages/translation/src/lang/en.ts

```diff
@@ -636,6 +635,38 @@ export default {
         },
       },
     },
+    dnsHoleSummary: {
+      name: "DNS Hole Summary",
+      description: "Displays the summary of your DNS Hole",
+      option: {
+        layout: {
+          label: "Layout",
+          option: {
+            row: {
+              label: "Horizontal",
+            },
+            column: {
+              label: "Vertical",
+            },
+            grid: {
+              label: "Grid",
+            },
+          },
+        },
+        usePiHoleColors: {
+          label: "Use Pi-Hole colors",
+        },
+      },
+      error: {
+        internalServerError: "Failed to fetch DNS Hole Summary",
```

Comment:
Use the error above when throwing, maybe add a variable interpolation to name the erroring widget / integration

Reply:
I don't think it makes sense to translate server errors that are shown in the console. And as it is it makes no sense to show the failed integration because it's defined in the options, right?